aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/AWSCognitoAuthPlugin.kt

```
@@ -717,21 +720,32 @@ class AWSCognitoAuthPlugin : AuthPlugin<AWSCognitoAuthServiceBehavior>() {
         }
     }
 
+    @OptIn(DelicateCoroutinesApi::class)
     override fun resetPassword(
         username: String,
         options: AuthResetPasswordOptions,
         onSuccess: Consumer<AuthResetPasswordResult>,
         onError: Consumer<AuthException>
     ) {
-        TODO("Not yet implemented")
+        val cognitoIdentityProviderClient = configureCognitoClients().cognitoIdentityProviderClient ?: return
```

Failing silently is not a good option.